Utilities/Get-SubscriptionUsage.ps1

```diff
@@ -203,6 +203,9 @@ foreach ($region in $allRegions)
     $currentRegionUsage =  Get-AzureRmVMUsage -Location $region
     Write-LogInfo "Get-AzureRmVMUsage -Location $region"
     $currentRegionAllowedCores = ($currentRegionUsage | Where-Object { $_.Name.Value -eq "cores"}).Limit
+    if (-not $currentRegionAllowedCores) {
```

Comment:
remove this addition

Reply:
@ader1990 , This check is needed because, for newly launched regions, it is not always guaranteed that new region is registered to our subscription.